WordPress/src/main/java/org/wordpress/android/ui/main/MySiteFragment.java

```diff
@@ -1049,4 +1121,113 @@ private void showActiveQuickStartTutorial() {
+                getString(R.string.quick_start_button_positive), new OnClickListener() {
+                    @Override public void onClick(View v) {
+                        if (showContinueQuickStartDialog) {
+                            ActivityLauncher.viewQuickStartForResult(getActivity());
+                        } else {
+                            mActiveTutorialPrompt = mySitePrompt;
+                            showActiveQuickStartTutorial();
+                        }
+                    }
+                });
+
+        mQuickStartTaskPromptSnackBar
+                .setNegativeButton(getString(R.string.quick_start_button_negative), new OnClickListener() {
+                    @Override public void onClick(View v) {
+                    }
+                });
+
+        mQuickStartTaskPromptSnackBar.show();
+        mQuickStartSnackBarWasShown = true;
+        incrementNumberOfTimesQuickStartDialogWasShown();
+    }
+
+    private void incrementNumberOfTimesQuickStartDialogWasShown() {
+        AppPrefs.setNumberOfTimesQuickStartDialogShown(AppPrefs.getNumberOfTimesQuickStartDialogShown() + 1);
+    }
+
+    private boolean shouldShowQuickStartTaskPrompt() {
+        return AppPrefs.getNumberOfTimesQuickStartDialogShown() <= MAX_NUMBER_OF_TIMES_TO_SHOW_QUICK_START_DIALOG
+               && !mQuickStartSnackBarWasShown && getPromptedQuickStartTask() != null;
+    }
+
+    /**
+     * Cycles through Quick Start tasks and picks returns a prompt information for the next unfinished one
+     */
+    private QuickStartMySitePrompts getNextQuickStartPrompt() {
+        for (QuickStartMySitePrompts quickStartMySitePrompt : QuickStartMySitePrompts.values()) {
+            if (!mQuickStartStore.hasDoneTask(AppPrefs.getSelectedSite(), quickStartMySitePrompt.getTask())) {
+                return quickStartMySitePrompt;
+            }
+        }
+        return null;
+    }
+
+    /**
+     * Returns a Quick Start task that is currently being prompted to the user with a Snackbar
+     */
+    private QuickStartTask getPromptedQuickStartTask() {
+        String stringValue = AppPrefs.getPromptedQuickStartTask();
+        QuickStartTask task = null;
+        if (!TextUtils.isEmpty(stringValue)) {
+            task = QuickStartTask.Companion.fromString(stringValue);
+        }
+
+        return task;
+    }
+
+    /**
+     * Records Quick Start task that is currently being prompted to the use with a Snackbar
+     */
+    private void setPromptedQuickStartTask(QuickStartTask task) {
+        if (task == null) {
+            AppPrefs.setPromptedQuickStartTask(null);
+        } else {
+            AppPrefs.setPromptedQuickStartTask(task.toString());
+        }
+    }
+
+    /**
+     * In order to show Quick Start snackbar in onResume but not during orientation change we maintain a flag that
+     * indicates that Snackbar was shown once.
+     */
+    private void markQuickStartPromptAsShownOnce() {
```

Comment:
Is the `markQuickStartPromptAsShownOnce()` method necessary or used more often in future changes?  It's only called once in `onActivityResult` and `mQuickStartSnackBarWasShown = true` is actually used in `showQuickStartDialogTaskPrompt(boolean)` without calling this method.

Reply:
Good catch! It's an artifact from times before :)